src/java/org/apache/cassandra/db/ReadExecutionController.java

```diff
@@ -50,6 +61,13 @@ public class ReadExecutionController implements AutoCloseable
     private final RepairedDataInfo repairedDataInfo;
     private long oldestUnrepairedTombstone = Long.MAX_VALUE;
 
+    /*
+     * Track bulk transfers involved in the read, so we can do read reconciliation.
+     * These come from the ViewFragment, not the SSTable read path, so bloom filters + short-circuiting SSTable scans
+     * will still include the total set of relevant bulk transfers.
+     */
```

Comment:
nit: If you use a JavaDoc comment here, you can do fun stuff like `{@link ColumnFamilyStore.ViewFragment}`...FWIW